README.md

```diff
@@ -81,17 +81,17 @@ The Telegram Bot does the followinng:
 2. Find out which facilities your NHIF card will cover in your county
 3. Find the nearest doctor or health facility
 
-### Installation
+**Installation**
 Make sure you have installed in your environment: `python-telegram-bot`
```

Comment:
Why the choice to have it as bold instead of headings? To note: headings actually allow automatic linking if one wants to reference a section specifically.

Reply:
It did not match the document's format. Will revert to headings.